### ISSUE TITLE
TemplateCtrl undefined when updating a template case

### ISSUE THREAD

**Issue by axpatito** [TheHive-Project/TheHive#519]
# Possible bug when updating a case template

### Request Type
Bug

### Work Environment

| Question              | Answer
|---------------------------|--------------------
| OS version (server)       | CentOS
| OS version (client)       | WIN10
| TheHive version / git hash   | 63f4f4522b1df9498cc6fb57f537e05109c8f89b
| Package Type              | From Source
| Browser type & version    | Chrome

### Problem Description
Saving the attached case template will upload fine, but updating it will cause a 400 error.

### Steps to Reproduce
1. Import the attached case template
2. Confirm correct case creationg
3. Attempt to update the case template

### Complementary information
400 Response for the case request:
```
{"tableName":"caseTemplate","type":"AttributeCheckingError","errors":[[{"name":"caseTemplate._version","value":{"type":"JsonInputValue","value":1},"type":"UnknownAttributeError","message":"Unknown attribute caseTemplate._version: {\"type\":\"JsonInputValue\",\"value\":1}"},{"name":"caseTemplate._id","value":{"type":"JsonInputValue","value":"AWJuEEGtVtbQGZMQkE3-"},"type":"UnknownAttributeError","message":"Unknown attribute caseTemplate._id: {\"type\":\"JsonInputValue\",\"value\":\"AWJuEEGtVtbQGZMQkE3-\"}"},{"name":"caseTemplate._routing","value":{"type":"JsonInputValue","value":"AWJuEEGtVtbQGZMQkE3-"},"type":"UnknownAttributeError","message":"Unknown attribute caseTemplate._routing: {\"type\":\"JsonInputValue\",\"value\":\"AWJuEEGtVtbQGZMQkE3-\"}"},{"name":"caseTemplate._parent","value":{"type":"JsonInputValue","value":null},"type":"UnknownAttributeError","message":"Unknown attribute caseTemplate._parent: {\"type\":\"JsonInputValue\",\"value\":null}"}]]}
```
[Case-Template__CS-Análisis_de_correo.txt](https://github.com/TheHive-Project/TheHive/files/1857535/Case-Template__CS-Analisis_de_correo.txt)

**Comment by mcvic1rj** [comment]
I believe this is related to #502. According to twitter 3.0.7 will be released tomorrow and that should fix this bug.

**Comment by saadkadhi** [comment]
Fixed in 3.0.7 which is publicly available.
